test_setup/sql/init_db_for_updating_a_childs_data.sql

```diff
@@ -26,6 +26,8 @@ UNLOCK TABLES;
 LOCK TABLES `AB_CARS_Children` WRITE;
 INSERT INTO `AB_CARS_Children` (`uuid`, `created_at`, `updated_at`, `properties`, `translations`, `Gender`, `No`, `Religion`, `Birthday`, `ID Number`, `ID Issue Date`, `Profile Photo`, `Fingerprintss`, `Type Received`, `ID Expire Date`, `First Name`, `Last Name`, `Home`, `Address`, `CARs Project20`) 
 VALUES ("0e41a300-4698-40c8-9c5f-f96ea2ceadf6", "2021-10-01 09:44:00", "2021-10-01 09:44:00", NULL, "[{\"language_code\":\"en\",\"Nickname\":\"Sato\",\"Race\":\"Human\",\"Nationality\":\"Japanese\",\"Time Received for\":\"05/04/1975 17:00 PM\"}]", "1507525545560", '1', "1507525546158", "1975-04-05", NULL, NULL, NULL, NULL, "1555919727844", NULL, "Satoshi", "Nakamoto", "696cfe80-8a3e-477f-a566-3c14dac1cc2d", NULL, "cb46b6b4-dc25-4915-80f7-c1febf26da4a");
+VALUES ("0e41a300-4698-40c8-9c5f-f96ea2ceadf4", "2021-10-01 09:44:00", "2021-10-01 09:44:00", NULL, "[{\"language_code\":\"en\",\"Nickname\":\"Ant\",\"Race\":\"Human\",\"Nationality\":\"Welsh\",\"Time Received for\":\"31/12/1937 18:00 PM\"}]", "1507525545560", '2', "1507525546012", "1937-12-31", NULL, NULL, NULL, NULL, "1555919727884", NULL, "Anthony", "Hopkins", "696cfe80-8a3e-477f-a566-3c14dac1cc2d", NULL, "cb46b6b4-dc25-4915-80f7-c1febf26da4a");
+VALUES ("0e41a300-4698-40c8-9c5f-f96ea2ceadf2", "2021-10-01 09:44:00", "2021-10-01 09:44:00", NULL, "[{\"language_code\":\"en\",\"Nickname\":\"Brit\",\"Race\":\"Human\",\"Nationality\":\"American\",\"Time Received for\":\"02/12/1981 19:00 PM\"}]", "1507525545633", '3', "1507525546012", "1981-12-02", NULL, NULL, NULL, NULL, "1555919727884", NULL, "Britney", "Spears", "696cfe80-8a3e-477f-a566-3c14dac1cc2d", NULL, "cb46b6b4-dc25-4915-80f7-c1febf26da4a");
```

Comment:
You can remove these since you made a new SQL for the test.

Reply:
Ok, yes I tried many ways. So I found the only way to get into with eq.